src/hotspot/share/nmt/nmtTreap.hpp

```diff
@@ -173,9 +170,9 @@ class Treap {
 #ifdef ASSERT
   void verify_self() {
     // A balanced binary search tree should have a depth on the order of log(N).
-    // We take the ceiling of log_2(N + 1) * 2.5 as our maximum bound.
+    // We take the ceiling of log_2(N + 1) * 3 as our maximum bound.
     // For comparison, a RB-tree has a proven max depth of log_2(N + 1) * 2.
-    const int expected_maximum_depth = ceil((log(this->_node_count+1) / log(2)) * 2.5);
+    const int expected_maximum_depth = ceil((log(this->_node_count+1) / log(2)) * 3);
```

Comment:
Why ln? why not log2? And then, why not one of the log2 methods in utilities/power_of_two.hpp?

Reply:
Nice, I'll switch to those, I just wasn't aware of them.